src/deploy/functions/prepareFunctionsUpload.ts

```diff
@@ -116,15 +114,9 @@ async function packageSource(options: Options, sourceDir: string, configValues:
 }
 
 export async function prepareFunctionsUpload(
-  runtimeConfig: backend.RuntimeConfigValues | undefined,
-  options: Options
+  sourceDir: string,
+  config: projectConfig.ValidatedSingle,
+  runtimeConfig: backend.RuntimeConfigValues | undefined
```

Comment:
Can use an optional argument

```suggestion
  runtimeConfig?: backend.RuntimeConfigValues
```